dnote package is broken

### Problem description

if you install dnote and install another package then an error will occur
![Screenshot_2022-11-02-16-23-21-062_com termux](https://user-images.githubusercontent.com/110718021/199439751-c6030763-427b-4db1-b21c-437b367c7d48.jpg)

dnote-server also has an error 
![Screenshot_2022-11-02-16-24-14-071_com termux](https://user-images.githubusercontent.com/110718021/199439789-91bad94e-b4c5-4ef6-9321-377f9ca6bc84.jpg)

and cannot use dnote
![Screenshot_2022-11-02-16-30-57-456_com termux](https://user-images.githubusercontent.com/110718021/199440479-944ba9f9-46d4-4959-b3bc-75ca3ba0862e.jpg)


### What steps will reproduce the bug?

- if you install dnote and install another package
- run dnote-server
- run dnote new


### What is the expected behavior?

run normally

### System information

termux-info:

```
Termux Variables:
TERMUX_API_APP__VERSION_NAME=0.50.1+9e72088
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP__AM_SOCKET_SERVER_ENABLED=true
TERMUX_APP__APK_PATH=/data/app/com.termux-OYvSoX5TrjLW20Fx0DQ3CQ==/base.apk
TERMUX_APP__APK_RELEASE=GITHUB
TERMUX_APP__FILES_DIR=/data/user/0/com.termux/files
TERMUX_APP__IS_DEBUGGABLE_BUILD=true
TERMUX_APP__IS_INSTALLED_ON_EXTERNAL_STORAGE=false
TERMUX_APP__PACKAGE_MANAGER=apt
TERMUX_APP__PACKAGE_NAME=com.termux
TERMUX_APP__PACKAGE_VARIANT=apt-android-7
TERMUX_APP__PID=2048
TERMUX_APP__SE_FILE_CONTEXT=u:object_r:app_data_file:s0:c157,c256,c512,c768
TERMUX_APP__SE_INFO=default:targetSdkVersion=28:complete
TERMUX_APP__SE_PROCESS_CONTEXT=u:r:untrusted_app:s0:c157,c256,c512,c768
TERMUX_APP__TARGET_SDK=28
TERMUX_APP__UID=10157
TERMUX_APP__USER_ID=0
TERMUX_APP__VERSION_CODE=118
TERMUX_APP__VERSION_NAME=0.118.0+20dee0e
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0+20dee0e
Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://packages-cf.termux.dev/apt/termux-main stable main
# root-repo (sources.list.d/root.list)
deb https://packages-cf.termux.dev/apt/termux-root root stable
Updatable packages:
All packages up to date
termux-tools version:
1.31.0
Android version:
9
Kernel build information:
Linux localhost 4.9.117+ #1 SMP PREEMPT Wed Sep 16 18:28:51 WIB 2020 armv7l Android
Device manufacturer:
Xiaomi
Device model:
Redmi 6A

```

Can reproduce, error I get on installation on an arm64 device is

```
Setting up dnote (1:0.12.0) ...
panic: initializing context: running migration: running migration: running 'create-fts-index': creating note_fts: no such module: fts5

goroutine 1 [running]:
main.main()
        /home/builder/.termux-build/dnote/src/pkg/cli/main.go:51 +0x5cc
panic: initializing context: running migration: running migration: running 'create-fts-index': creating note_fts: no such module: fts5

goroutine 1 [running]:
main.main()
        /home/builder/.termux-build/dnote/src/pkg/cli/main.go:51 +0x5cc
panic: initializing context: running migration: running migration: running 'create-fts-index': creating note_fts: no such module: fts5

goroutine 1 [running]:
main.main()
        /home/builder/.termux-build/dnote/src/pkg/cli/main.go:51 +0x5cc
dpkg: error processing package dnote (--configure):
 installed dnote package post-installation script subprocess returned error exit status 2
Errors were encountered while processing:
 dnote
E: Sub-process /data/data/com.termux/files/usr/bin/dpkg returned an error code (1)
```

--- a/packages/dnote/build.sh
+++ b/packages/dnote/build.sh
@@ -3,6 +3,7 @@ TERMUX_PKG_DESCRIPTION="A simple command line notebook for programmers"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="Ravener <ravener.anime@gmail.com>"
 TERMUX_PKG_VERSION=1:0.12.0
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/dnote/dnote/archive/refs/tags/cli-v${TERMUX_PKG_VERSION:2}.tar.gz
 TERMUX_PKG_SHA256=2b2c276f3b381a853e155ed2eab7007c5b5f2ec927b3d882d5605f2650b15085
 TERMUX_PKG_BUILD_IN_SRC=true
@@ -16,23 +17,15 @@ termux_step_pre_configure() {
 
 termux_step_make() {
 	cd "$TERMUX_PKG_SRCDIR"
-	go build -o dnote -ldflags "-X main.versionTag=${TERMUX_PKG_VERSION:2}" pkg/cli/main.go
+	go build -o dnote -ldflags "-X main.versionTag=${TERMUX_PKG_VERSION:2}" -tags fts5 pkg/cli/main.go
 }
 
 termux_step_make_install() {
 	install -Dm700 $TERMUX_PKG_SRCDIR/dnote $TERMUX_PREFIX/bin/dnote
-	install -Dm644 /dev/null "${TERMUX_PREFIX}/share/bash-completion/completions/dnote.bash"
-	install -Dm644 /dev/null "${TERMUX_PREFIX}/share/zsh/site-functions/_dnote"
-	install -Dm644 /dev/null "${TERMUX_PREFIX}/share/fish/vendor_completions.d/dnote.fish"
-}
-
-termux_step_create_debscripts() {
-	cat <<- EOF > ./postinst
-		#!${TERMUX_PREFIX}/bin/sh
-		dnote completion bash > ${TERMUX_PREFIX}/share/bash-completion/completions/dnote.bash
-		dnote completion zsh > ${TERMUX_PREFIX}/share/zsh/site-functions/_dnote
-		dnote completion fish > ${TERMUX_PREFIX}/share/fish/vendor_completions.d/dnote.fish
-	EOF
+	install -Dm600 "$TERMUX_PKG_SRCDIR"/pkg/cli/dnote-completion.bash \
+		"$TERMUX_PREFIX"/share/bash-completion/completions/dnote
+	install -Dm600 "$TERMUX_PKG_SRCDIR"/pkg/cli/dnote-completion.zsh \
+		"$TERMUX_PREFIX"/share/zsh/site-functions/_dnote
 }
 
 termux_step_install_license() {

